tests/unit/s2n_handshake_test.c

```diff
@@ -118,7 +171,19 @@ int test_cipher_preferences(struct s2n_config *server_config, struct s2n_config
             expect_failure = 1;
         }
 
-        /* Craft a cipher preference with a cipher_idx cipher */
+#if 0
+        fprintf(stderr, "Testing %s in %s mode, expect_failure=%d\n", expected_cipher->name,
+                test_type == TEST_TYPE_SYNC ? "synchronous" : "asynchronous", expect_failure);
+#endif
+
```

Comment:
did you want to keep this?

wondering if we should have a debug print macro which defaults to nothing:

```
#if !defined(S2N_TEST_DEBUG)
#  define S2N_TEST_PRINT(x) 
#else
#  define S2N_TEST_PRINT(x)  x
#endif
```

so we don't have thiee `#if 0` statements committed and can enable more verbose test output on demand.  thoughts?

Reply:
Yeah, I do want to keep it.

Was thinking of adding some mechanism to print stuff into temporary buffer and write it to `stderr` on test failure. But your option is simpler, so I can probably do it now.

Reply:
https://github.com/awslabs/s2n/pull/1979